### PR TITLE
deposit wallet

### PR DESCRIPTION
**Pull request by ferdigokdemir**

### CODE REVIEW

**Comment by radek1st** [comment]
@sonu-sonam please have a look

File: pages/deposit.js [hunk 1]
```diff
@@ -681,12 +712,17 @@ class Deposit extends React.Component {
                   <div className="col-sm-12 col-md-12 col-lg-7">
                     <div className="qr-to-address p-3 p-lg-4 mt-4 mt-lg-5">
                       <p className="to-addres">
-                        Please enter valid {this.state.createdTransaction.toCurrency} address
+                        Please enter valid{" "}
+                        {this.state.createdTransaction.toCurrency} address
                       </p>
 
                       <Formik
```
Comment (sonu-sonam) [inline pages/deposit.js:734]: According to Charlie's new flow design, the wallet address form on 'transfer' step should have 'Back to address page' button and redirect back to 'wallet' step.

Comment (radek1st) [reply pages/deposit.js:734]: @charlierelaypay

Comment (charlierelaypay) [reply pages/deposit.js:734]: @sonu-sonam @radek1st I guess the designer wanted to put this if a user wants to change the Address on Transfer page. We might kill that button.
We should be doing all Wallet entry, validity controls on Wallet Step

**Pull request merged**